tests/npltests/npl_pod_test.go

```diff
@@ -12,7 +12,7 @@
 * limitations under the License.
 */
 
-package hostnameshardtests
+package npltests
```

Comment:
could you please rebase once, i remember doing this change and some addition regarding CRDClientset in a previous PR

Reply:
I think that PR is not merged yet :-)

Reply:
ah, i'll rebase then :) thanks